impl/src/main/java/org/pentaho/di/trans/dataservice/optimization/OptimizationImpactInfo.java

```diff
@@ -22,7 +22,7 @@
 
 package org.pentaho.di.trans.dataservice.optimization;
 
-import com.google.common.base.Objects;
+import com.google.gwt.thirdparty.guava.common.base.Objects;
```

Comment:
why are we using the GWT version of the file `com.google.gwt.thirdparty.guava.common.base.Objects` instead of using the `com.google.common.base.MoreObjects`? 

It seems like other changes are swapping `com.google.common.base.Objects` with `com.google.common.base.MoreObjects`

Reply:
fixed with https://github.com/pentaho/pdi-dataservice-server-plugin/commit/2c34eb09172284f7c60294657f4601b84e8fdc0c